src/main/java/Storage.java

```diff
@@ -0,0 +1,19 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Storage {
+    public static void createAndWrite(String path, TaskList listOfTasks) {
+        try {
+            File savedTasks = new File(path);
```

Comment:
Perhaps savedTask would be better as it's a singular File instead of a List of Files

https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions